bridge/irc_listener.go

```diff
@@ -52,58 +56,58 @@ func (i *ircListener) nickTrackNick(event *irc.Event) {
 		i.bridge.ircManager.puppetNicks[newNick] = con
 		delete(i.bridge.ircManager.puppetNicks, oldNick)
 	}
+}
+
+func (i *ircListener) OnNickRelayToDiscord(event *irc.Event) {
+	// Ignored hostmasks
+	if i.bridge.ircManager.isIgnoredHostmask(event.Source) {
+		return
+	}
+
+	oldNick := event.Nick
+	newNick := event.Message()
+
+	msg := IRCMessage{
+		Username: "",
+		Message:  fmt.Sprintf("_%s changed nick to %s_", oldNick, newNick),
```

Comment:
This is gonna be configurable in #76 right?

Reply:
It is, I'm not too attached to the current format being a default either. Was more what is actually used, up till I started working on formatting.

Reply:
I thought I added "their" in, before, but I apparently undid that somewhere... will refix that.